Game/Scripts/UIGameManager.cpp

```diff
@@ -92,13 +95,17 @@ void UIGameManager::MenuIsOpen()
 {
 	if (menuIsOpen)
 	{
+		manager->GetComponent<EnemiesManager>()->PauseEnemies(true);
 		mainMenuObject->Enable();
 		hudCanvasObject->Disable();
+		App->GetModule<ModulePlayer>()->GetPlayer()->GetComponent<PlayerManagerScript>()->FullPausePlayer(true);
```

Comment:
at the end of the else-if block:
```suggestion
		App->GetModule<ModulePlayer>()->GetPlayer()->GetComponent<PlayerManagerScript>()->FullPausePlayer(menuIsOpen);
```